llvm/test/ThinLTO/X86/ctxprof.ll

```diff
@@ -47,18 +47,21 @@
 ; NOPROF-1-NOT: m2_f1()
 ; NOPROF-2-NOT: m1_f1()
 ;
-; The run with workload definitions - same other options.
+; The run with workload definitions - same other options. We do need to re-generate the .bc
+; files, to include instrumentation.
```

Comment:
How did this work before this change?

Reply:
These functions have no edges. Before, the validation had to do with edges, now we insist all counters are defined.